eradiate/radprops/rad_profile.py

```diff
@@ -393,6 +394,126 @@ def to_dataset(self, spectral_ctx=None):
+_DEFAULT_SIGMA_T = ureg.Quantity(
+    np.array(
+        [
+            1.09518903e-02,
+            9.92737262e-03,
+            8.97815462e-03,
+            8.10027197e-03,
+            7.28988381e-03,
+            6.54327153e-03,
+            5.85683756e-03,
+            5.22710408e-03,
+            4.65071185e-03,
+            4.12441899e-03,
+            3.64509969e-03,
+            3.16532552e-03,
+            2.70512979e-03,
+            2.31195811e-03,
+            1.97603132e-03,
+            1.68899981e-03,
+            1.44373407e-03,
+            1.23414607e-03,
+            1.05503681e-03,
+            9.01966264e-04,
+            7.69652991e-04,
+            6.55494081e-04,
+            5.58703489e-04,
+            4.76573170e-04,
+            4.06827441e-04,
+            3.47552351e-04,
+            2.97136843e-04,
+            2.54223707e-04,
+            2.17668653e-04,
+            1.86506127e-04,
+            1.59920743e-04,
+            1.37223387e-04,
+            1.17529727e-04,
+            1.00313955e-04,
+            8.57850609e-05,
+            7.34985548e-05,
+            6.30875403e-05,
+            5.42484772e-05,
+            4.67296802e-05,
+            4.03220044e-05,
+            3.48512844e-05,
+            3.01721820e-05,
+            2.61631670e-05,
+            2.27224151e-05,
+            1.97644462e-05,
+            1.72173641e-05,
+            1.50205848e-05,
+            1.31428283e-05,
+            1.16061428e-05,
+            1.02495276e-05,
+            9.05183520e-06,
+            8.00149759e-06,
+            7.13443859e-06,
+            6.35402301e-06,
+            5.65233552e-06,
+            5.02211492e-06,
+            4.45670599e-06,
+            3.95001423e-06,
+            3.49646369e-06,
+            3.09095752e-06,
+            2.72884131e-06,
+            2.40586864e-06,
+            2.11816924e-06,
+            1.86221916e-06,
+            1.63481317e-06,
+            1.43303884e-06,
+            1.25425286e-06,
+            1.09605862e-06,
+            9.56285783e-07,
+            8.32971089e-07,
+            7.24340787e-07,
+            6.28794196e-07,
+            5.43569783e-07,
+            4.68756761e-07,
+            4.03703417e-07,
+            3.47207672e-07,
+            2.98206611e-07,
+            2.55761392e-07,
+            2.19043788e-07,
+            1.87323925e-07,
+            1.59959331e-07,
+            1.36385248e-07,
+            1.16105733e-07,
+            9.86858710e-08,
+            8.37447766e-08,
+            7.09492031e-08,
+            5.97393986e-08,
+            5.00165885e-08,
+            4.18836096e-08,
+            3.50797824e-08,
+            2.93870330e-08,
+            2.46196683e-08,
+            2.06121737e-08,
+            1.72454030e-08,
+            1.44203783e-08,
+            1.20527715e-08,
+            1.00710400e-08,
+            8.41472640e-09,
+            7.03205639e-09,
+            5.87861565e-09,
```

Comment:
Not sure if hard-coding this in the Python module is best; maybe it should go to a CSV or NetCDF file in the data submodule?

Reply:
Using a NetCDF file seems more appropriate indeed.